src/theme/style.scss

```diff
@@ -1,29 +1,81 @@
 @import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;700&display=swap");
+$mycolor: "#83CE27";
 
 body {
-  font-family: "Montserrat", "sans-serif";
+  font-family: "serif", "sans-serif";
+  background-color: "#83CE27";
 }
 
+svg{
+ display: flex;
+ position: relative;
+}
 .title {
   font-size: 2.25em;
+  font-weight: bold;
+  letter-spacing: 2px;
   text-transform: uppercase;
 }
 
-.welcome-content {
+#root{
+  margin: 0px;
```

Comment:
If you ever need to use 0 - and be careful setting it on roots - you should use `0` instead of `0px`, the unit doesn't matter and `0` is faster in some browsers.